docs/src/extend/shareable-configs-deprecated.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Share Configurations (Deprecated)
+---
+
+::: warning
+This documentation is for shareable configs using the deprecated eslintrc configuration format. [View the updated documentation](shareable-configs).
+:::
+
+To share your ESLint configuration, create a **shareable config**. You can publish your shareable config on [npm](https://www.npmjs.com/) so that others can download and use it in their ESLint projects.
+
+This page explains how to create and publish a shareable config.
+
+## Creating a Shareable Config
+
+Shareable configs are simply npm packages that export a configuration object or array. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would.
```

Comment:
```suggestion
Shareable configs are simply npm packages that export a configuration object. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would.
```

I'm guessing this was mistakenly added here instead of in the other file.

Reply:
No, this was intentional. I'm pointing out everywhere that you can either provide an object or an array.